@re-/assert/test/metaTests/testHelpers.ts

```diff
@@ -0,0 +1,110 @@
+import { strict } from "node:assert"
+import { fromHere, readJson, shell } from "@re-/node"
+import {
+    ExpressionStatement,
+    Project,
+    SourceFile,
+    SyntaxKind,
+    VariableStatement
+} from "ts-morph"
+
+const EXPECTED_RESULTS_JSON_PATH = fromHere("expectedResults.json")
+const PATH_TO_ASSERTIONS_DIR = fromHere(".reassert")
+
+const getExpressionStatements = (
+    functionName: string,
+    varStatements: VariableStatement[]
+) =>
+    varStatements
+        .find((statement) => statement.getText().includes(functionName))
+        ?.getDescendantsOfKind(SyntaxKind.ExpressionStatement)
+
+const getTestComment = (statement: ExpressionStatement, text: string) => {
+    const range = statement.getLeadingCommentRanges()
+    return text
+        .slice(range[0].getPos(), range[0].getEnd())
+        .replace("//", "")
+        .trim()
+}
+export const cleanUpSourceFile = async (sf: SourceFile, template: string) => {
+    sf.replaceWithText(template)
+    await sf.save()
```

Comment:
So I think this save() call might be the source of the `async` infection that has spread throughout our tests and utility code here. Unless I'm missing another `async` call that's not so easy to replace, there is a `sourceFile.saveSync()` you can use here, which would allow you to make all of the tests and utilities synchronous.